lib/gui/components/safe-webview.js

```diff
@@ -198,26 +199,29 @@ class SafeWebview extends react.PureComponent {
    * @param {Event} event - event object
    *
    * @example
+   *
+   * // In the webview
    * console.log({
-   *   type: 0,
-   *   message: 'Hello, World!'
+   *   command: 'error',
+   *   data: 'Good night!'
    * });
+   *
+   * // or with robot
+   * robot.printError('Good night!');
```

Comment:
Oh, I don't mean through injection, more like decoupling `robot` from Etcher into its own module, but I guess if/until then I'll just remove this example.